tests/FakeItEasy.Tests/Core/FakeManagerTests.cs

```diff
@@ -342,62 +339,14 @@ public void Should_return_hash_code_of_fake_manager_when_GetHashCode_has_not_bee
             this.Should_set_default_return_value_when_object_method_has_not_been_configured(typeof(object).GetMethod("GetHashCode"), x => x.GetHashCode());
         }
 
-        [Fact]
-        public void Should_return_false_for_equals_method_when_fake_managers_are_not_the_same()
-        {
-            // Arrange
-            var interceptingProxy = new object();
-            var interceptingProxyManager = new FakeManager(typeof(int), 0, this.fakeManagerAccessor);
-            this.SetManager(interceptingProxy, interceptingProxyManager);
-            var proxyPassedToEquals = new object();
-            this.SetManager(proxyPassedToEquals, new FakeManager(typeof(int), 0, this.fakeManagerAccessor));
-
-            var equalsMethod = typeof(object).GetMethod("Equals", new[] { typeof(object) });
-
-            var interceptedCall = A.Fake<IInterceptedFakeObjectCall>();
-            A.CallTo(() => interceptedCall.Method).Returns(equalsMethod);
-            A.CallTo(() => interceptedCall.FakedObject).Returns(proxyPassedToEquals);
-            A.CallTo(() => interceptedCall.Arguments).Returns(new ArgumentCollection(new object[] { proxyPassedToEquals }, interceptedCall.Method));
-
-            // Act
-            ProcessFakeObjectCall(interceptingProxyManager, interceptedCall);
-
-            // Assert
-            A.CallTo(() => interceptedCall.SetReturnValue(false)).MustHaveHappened();
-        }
-
-        [Fact]
-        public void Should_return_true_for_equals_method_when_fake_managers_are_the_same()
-        {
-            // Arrange
-            var interceptingProxy = new object();
-            var interceptingProxyManager = new FakeManager(typeof(int), 0, this.fakeManagerAccessor);
-            this.SetManager(interceptingProxy, interceptingProxyManager);
-            var proxyPassedToEquals = new object();
-            this.SetManager(proxyPassedToEquals, interceptingProxyManager);
-
-            var equalsMethod = typeof(object).GetMethod("Equals", new[] { typeof(object) });
-
-            var interceptedCall = A.Fake<IInterceptedFakeObjectCall>();
-            A.CallTo(() => interceptedCall.Method).Returns(equalsMethod);
-            A.CallTo(() => interceptedCall.FakedObject).Returns(proxyPassedToEquals);
-            A.CallTo(() => interceptedCall.Arguments).Returns(new ArgumentCollection(new object[] { proxyPassedToEquals }, interceptedCall.Method));
-
-            // Act
-            ProcessFakeObjectCall(interceptingProxyManager, interceptedCall);
-
-            // Assert
-            A.CallTo(() => interceptedCall.SetReturnValue(true)).MustHaveHappened();
-        }
-

```

Comment:
These started failing, because we don't use the faked `FakeManagerAccessor`. Moved to specs (which you saw above), as near as I was able.